roles/pulp3-redis/tasks/main.yml

```diff
@@ -4,19 +4,12 @@
   tags:
     - always
 
-- name: Add redis repository
-  apt_repository:
-    repo: ppa:chris-lea/redis-server
-  become: yes
-  when: ansible_distribution == 'Ubuntu'
-
 - block:
 
-    - name: Install requirements
-      package:
-        name: "{{ item }}"
-        state: present
-      with_items: "{{ pulp_preq_packages }}"
```

Comment:
Are you sure it's safe to drop this? At the moment, we always use the roles together, but in theory they should be able to be used separately.